lib/vacations.ex

```diff
@@ -0,0 +1,168 @@
+defmodule GlassFactoryApi.Vacations do
+  @moduledoc """
+  Provides functions to get vacations of all or a specific user.
+  """
+
+  alias GlassFactoryApi.ApiClient
+  alias GlassFactoryApi.Vacations.Vacation
+
+  @doc """
+  Return a tuple with `:ok` atom and an list of vacations.
+  or a tuple with `:error` and a string with the error description.
+
+  It's possible apply some filters, like user_id, start and end to limit dates, and vacation_type
+  using the query_string param.
+
+  ## Arguments
+
+  - query_string - options to filter the vacations.
```

Comment:
I think we need to indent this too. Let's check ex_doc's output.

Reply:
@britto

Here a print screen from `get_vacation` function, I like this way, shows to me that options are to `query_string` param, but I could see your opinion too. WDYT?

<img width="948" alt="Captura de Tela 2019-12-02 às 11 42 49" src="https://user-images.githubusercontent.com/13913773/69968381-0d410a80-14f9-11ea-8bb5-946ad56eea3a.png">

Reply:
Yes, it works without indentation. However, indenting bullet lists with 2 spaces seems to be a standard adopted by the Elixir core team, so I'd follow them. Examples:

https://github.com/elixir-lang/elixir/blob/055526057ac9992cc1df87441357c4e56d45235d/lib/elixir/lib/string.ex#L327-L333
https://github.com/elixir-lang/elixir/blob/055526057ac9992cc1df87441357c4e56d45235d/lib/elixir/lib/string.ex#L75-L79